runtime/gc_base/GCExtensions.hpp

```diff
@@ -149,14 +149,17 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	UDATA deadClassLoaderCacheSize;
 #endif /*defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING) */
 
-
+	bool statupPhaseFinished; /**< Used to indicate if the JIT startup phase is finished */
```

Comment:
this is not used anymore - we should probably remove it?

Reply:
Right... I'm not sure whether to keep it or not. On one hand, we don't currently use it, but on the other, I think this might be a useful flag for other scenarios in the future. Up to you if we keep it